.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,17 +1,17 @@
 ---
 name: "\U0001F41E Bug Report"
 about: Help us improve and fix existing issues
-title: "[BUG]"
+title: '[BUG]'
```

Comment:
These changes are from another PR, did you rebase from master? I think it's ok though because these changes look correct.

Reply:
Huh, not sure how that happened. Did a git checkout -b "my_branch_name" while on master

Reply:
I'm not sure either, did you do a `git pull` before checking out your feature branch?